plugin.json

```diff
@@ -109,6 +109,13 @@
                         "value": "disable"
                     }
                 ]
+            },
+            {
+                "key": "EnableWebhookEventLogging",
+                "display_name": "Enable Webhook Event Logging:",
+                "type": "bool",
+                "help_text": "Allow the plugin to log the event from webhook.",
+                "default": true
```

Comment:
Should the setting be disabled by default? I could image it being to spammy for a default setting.

Reply:
Sure , ll change to false